src/meta/src/controller/mod.rs

```diff
@@ -60,16 +60,17 @@ pub struct SqlMetaStore {
     pub conn: DatabaseConnection,
 }
 
+pub const IN_MEMORY_STORE: &str = "sqlite::memory:";
```

Comment:
Shall we specify a name for the instance to avoid (potentially) being conflict with other usages?

https://arc.net/l/quote/dpqhqzhz

Reply:
It's solely used for unit tests, and upon local testing, it appears conflict-free. Let's keep it unchanged. It should be useful in simulation tests.